osu.Game/Graphics/Backgrounds/Triangles.cs

```diff
@@ -99,16 +99,18 @@ protected virtual Triangle CreateTriangle()
 
             const float size = 100;
 
-            return new Triangle
+            return new EquilateralTriangle
             {
                 Origin = Anchor.TopCentre,
+

```

Comment:
Yup, I like to split these declarations into the following:
1) Origins/Anchors
2) Sizing/positioning
3) Other single-line properties (colour, border, depth, etc).
4) Multi-line properties (edgeeffect)
5) Children =...

Just helps with readability imo, so I don't personally enforce it.